src/transformState.js

```diff
@@ -6,6 +6,29 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    const { type, extraData, keysToRemove } = action;
```

Comment:
🔥

Reply:
what its meen?
"IT'S GREAT!" or "ooooh, f**k..."